Cookbook/Technical-Documents/UnitTestingViewModels.md

```diff
@@ -0,0 +1,51 @@
+## Unit testing view models
+
+### What do we test in the view model unit tests?
+
+- view model state changes
+- routing events
+- analytics events
+
+### Creating a new test case
+
+To create a test case for a view model use an Xcode template called `BentoViewModelTests` and provide the view model name (without `ViewModel` suffix). This template will create a basic structure for the test case that you will need to adjust to your specific implementation.
```

Comment:
```suggestion
To create a test case for a view model use the Xcode template `BentoViewModelTests` and provide the view model name (without `ViewModel` suffix). This template will create a basic structure for the test case that you will need to adjust to your specific implementation.

E.g: your feature is called `MyFeature`, and is has a `MyFeatureViewModel`. You just need to name the file `MyFeature`, the template will automatically add the proper suffix making it `MyFeatureViewModelTests`.
```

Reply:
@diegopetrucci 
> ... and is has ... 

... and *it* has ... would you mind if I edit your suggestion?

Reply:
also before the "and" no comma is needed.

Reply:
Sure big man, feel free

Reply:
just to be clear: I'll do it just when it's a typo in a suggestion. Otherwise it's a rabbit hole. 😉